application/src/main/java/org/togetherjava/tjbot/commands/ReloadCommand.java

```diff
@@ -75,12 +75,11 @@ public void onButtonClick(ButtonClickEvent event, List<String> idArgs) {
                     restActions.add(getGlobalCommandUpdateRestAction(event, commands));
 
                     // * Triggers all RestActions, when they're all finished the message gets send.
-                    RestAction.allOf(restActions).queue(updatedCommands -> {
-                        event.getHook()
+                    RestAction.allOf(restActions)
+                        .queue(updatedCommands -> event.getHook()
```

Comment:
did spotless do this formatting?

Reply:
I believe so, I didn't even open the ReloadCommand class myself